src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -215,7 +215,7 @@ private function getLegend()
         $legend = [];
         foreach (self::status() as $name) {
             $legend[$name] = [
-                'label' => $this->translator->trans('service.overview.legend.'.$name),
+                'label' => $this->getSanitizedTranslation('service.overview.legend.'.$name),
```

Comment:
I think it will be valuable in terms of defense in depth.

Reply:
I don't think it would have done any damage to kept it as is but I'm fine to remove it from the legends because I did some additional testing if the legends were thoroughly escaped and they were.